pkg/multiplexer/multiplexer_test.go

```diff
@@ -89,7 +89,7 @@ func TestMultiplexer_Start(t *testing.T) {
 	}()
 	const muxServer = "127.0.0.1:1235"
 
-	mux := New(l.Addr().String(), "1235", message.EchoMessageReader{})
+	mux := New(l.Addr().String(), "1235", message.EchoMessageReader{}, 0, 10*time.Second)
```

Comment:
Could you please reduce this a bit to say 5 seconds? That would still be enough to verify that the setting is effective but would slow down automated tests a little less.

Reply:
The 10 Seconds is the timeout. The delay is zero. I reduced it to 5 Seconds anyways. Should not hurt.